src/runtime-rs/crates/hypervisor/src/ch/inner.rs

```diff
@@ -156,12 +157,34 @@ impl Persist for CloudHypervisorInner {
         _hypervisor_args: Self::ConstructorArgs,
         hypervisor_state: Self::State,
     ) -> Result<Self> {
+        let mut capabilities = Capabilities::new();
+        capabilities.set(
+            CapabilityBits::BlockDeviceSupport
+                | CapabilityBits::BlockDeviceHotplugSupport
+                | CapabilityBits::FsSharingSupport
+                | CapabilityBits::HybridVsockSupport,
+        );
```

Comment:
This duplicates the code we have in `new()` so can we either create a new caps setup method to be called from `new()` and `restore()`, or somehow extract it from the args?

Reply:
@jodh-intel,  there is an existing capabilities() function for CloudHypervisorInner. I made use of that instead. Which also makes me wonder if we really want to have a field called _capabilites at all. Would be just better to make use of the function instead. Wdyt? Could be addressed in a separate PR of course as the change would be unrelated.

Reply:
> @jodh-intel, there is an existing capabilities() function for CloudHypervisorInner. I made use of that instead.

+1.

> Which also makes me wonder if we really want to have a field called _capabilites at all. Would be just better to make use of the function instead. Wdyt? Could be addressed in a separate PR of course as the change would be unrelated.

I agree. And I also spotted that we're setting caps in `CloudHypervisorInner::new()`, but that code could also just call `self.capabilities()` to avoid the duplication.